app/src/main/java/org/jellyfin/androidtv/ui/AlphaPicker.kt

```diff
@@ -0,0 +1,42 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.Button
+import android.widget.LinearLayout
+import org.jellyfin.androidtv.R
+
+class AlphaPicker(
+	context: Context,
+	attrs: AttributeSet?
+) : LinearLayout(context, attrs) {
+	var onAlphaSelected: (s: String) -> Unit = {}
+
+	init {
+		removeAllViews()
+
+		val letters = "#${resources.getString(R.string.byletter_letters)}"
+		letters.toCharArray().forEach {
+			// NOTE: We must use a Button here and NOT AppCompatButton due to focus issues on Fire OS
+			val button = Button(context)
+			button.text = it.toString()
+			button.setOnClickListener { _ ->
+				this.onAlphaSelected(it.toString())
+			}
+
+			addView(button)
+		}
+	}
+
+	fun focus(text: String?) {
```

Comment:
Wouldn't it make more sense to accept a `Char` here?

Reply:
Also makes more sense for it to be not-null I guess

Reply:
Probably... I switched to only using Chars in the class and added null checks on the Java side.